client/src/App.sass.d.ts

```diff
@@ -0,0 +1,11 @@
+// This file is automatically generated.
```

Comment:
Looks like there's going to be one generated file per component that uses sass? In that case, this might be the push we needed to break up our src/ dir into subdirectories 😃 is it possible to change the output dir for these generated files?

Reply:
I briefly looked into this. It's pretty common to have a src/types/ directory but I haven't been able to figure out how to extract that to a different directory (at least using typings-for-css-modules-loader) :/

Hopefully we can move this to a more graceful solution in the future.